pyanaconda/modules/localization/localization.py

```diff
@@ -200,5 +191,65 @@ def install_with_tasks(self):
         :returns: list of installation tasks
         """
         return [
-            LanguageInstallationTask(sysroot=conf.target.system_root, lang=self.language)
+            LanguageInstallationTask(
+                sysroot=conf.target.system_root,
+                lang=self.language
+            ),
+            KeyboardInstallationTask(
+                sysroot=conf.target.system_root,
+                x_layouts=self.x_layouts,
+                switch_options=self.switch_options,
+                vc_keymap=self.vc_keymap
+            )
         ]
+
+    def populate_missing_keyboard_configuration_with_task(self):
+        """Get missing keyboard configuration by conversion.
+
+        :returns: a task converting the configuration
+        """
+        if not self.x_layouts and not self.vc_keymap:
+            self.vc_keymap = DEFAULT_KEYBOARD
```

Comment:
Shouldn't we wait and set this property later with `update_settings_from_task`?

Reply:
Following on our discussion, finally I decided to move the default setting into the task:
https://github.com/rhinstaller/anaconda/pull/2217/commits/d462bc9a70b5db294521a371ae988a46747f63c8